cookiesales.js

```diff
@@ -0,0 +1,216 @@
+'use strict'
+
+//Create Store's Object
+var firstAndPike = {
+  ownersName: 'Oscar Cauich',
+  address: '1100 Sunset Bldv NE',
+  city:'Renton',
+  state: 'WA',
+  zipCode: 98056,
+  storeHours: ['6am', '7am','8am','9am','10am','11am','12pm','1pm','2pm','3pm','4pm','5pm','6pm','7pm','8pm'],
+  hourlyCookieSales: [],
+  phoneNumber: 2063885908,
+  hourlyMinCustomer: 23,
+  hourlyMaxCustomer: 65,
+  avergHourlyCookieSales: 6.3,
+  storeName: '1st and Pike',
+
+  forcastRandomCustomerPerHour: function() {
+    return Math.ceil(Math.random() * (this.hourlyMaxCustomer / this.hourlyMinCustomer));
+  },
+
+  hourlyStoreSales: function() {
+    for (var i = 0; i < this.storeHours.length; i++){
+      this.hourlyCookieSales.push(Math.ceil(Math.random()*(this.avergHourlyCookieSales * this.forcastRandomCustomerPerHour())))[i];
+      // console.log(this.hourlyCookieSales);
+    }
+  },
+}
+
+var getStoreHours = firstAndPike.storeHours;
```

Comment:
You don't need this var definition.  You are assigning it to an object property that you can call whenever you use this var instead.